tpm2tools/pcr.go

```diff
@@ -67,6 +68,77 @@ func ReadPCRs(rw io.ReadWriter, sel tpm2.PCRSelection) (*proto.Pcrs, error) {
 	return &pl, nil
 }
 
+// SealCurrent represent current PCRs states.
+type SealCurrent struct{ tpm2.PCRSelection }
+
+// SealTarget predicted sealing target PCRs.
+type SealTarget struct{ *proto.Pcrs }
+
+// SealingOpt will return a set of target PCRs when sealing.
+type SealingOpt interface {
+	PCRsForSealing(rw io.ReadWriter) (*proto.Pcrs, error)
+}
+
+// PCRsForSealing read from TPM and return the selected PCRs.
```

Comment:
No need for these member comments (here and below).

Reply:
If removed, go-linter will complain:
`exported method SealCurrent.PCRsForSealing should have comment or be unexported`